libs/contrib/Data/SortedSet.idr

```diff
@@ -31,5 +29,26 @@ export
 toList : SortedSet k -> List k
 toList (SetWrapper m) = map (\(i, _) => i) (Data.SortedMap.toList m)
 
-implementation Foldable SortedSet where
```

Comment:
I was under the impression that this was the new proper style, am I mistaken in this?